autoload/snipe/core.vim

```diff
@@ -140,7 +140,6 @@ endfunction
 function! s:GetInput(message) " {{{
   echo a:message
   let ord = getchar()
-  normal :<c-u>
```

Comment:
The purpose of this line is to remove the prompts after the user has entered it. With this change, if you initiate a jump and cancel, the prompt remains.

I think we need to understand the undo tree a bit more. I bet operator-pending motions aren't the only places where snipe functions have unexpected side effects on it.

Reply:
Aha, that was my impression. I don't really write viml; sorry about that. I will keep this line removed in my local copy for now, but we can close this PR if you think we can get a better solution in the future.

Reply:
No problem. I think we should focus on some more reproducible test cases on #24 before getting this going. Thanks for the submission!